src/doc/en/developer/packaging.rst

```diff
@@ -1059,36 +1059,47 @@ You can skip the manual downloading of the upstream tarball by using
 the additional argument ``--upstream-url``.  This command will also
 set the ``upstream_url`` field in ``checksums.ini`` described above.
 
-For Python packages available from PyPI, you can use::
+For Python packages available from PyPI, use a PURL (Package URL,
+see `PEP 725 <https://peps.python.org/pep-0725/#concrete-package-specification-through-purl>`_)::
 
-    [alice@localhost sage]$ sage --package create scikit_spatial --pypi   \
-                                             --type optional
+    [alice@localhost sage]$ ./sage --package create pkg:pypi/scikit-spatial \
+                                               --type optional
 
-This automatically downloads the most recent version from PyPI and also
-obtains most of the necessary information by querying PyPI. In particular,
-the ``SPKG.rst`` file is created as a copy of the package's README file.
+An equivalent command uses the SPKG name of the new package::
 
+    [alice@localhost sage]$ ./sage --package create scikit_spatial --pypi   \
+                                               --type optional
 
-The ``dependencies`` file may need editing (watch out for warnings regarding
-``--no-deps`` that Sage issues during installation of the package!).
+Either of these two commands automatically downloads the most recent version
+from PyPI and also obtains most of the necessary information by querying PyPI.
+In particular, the ``SPKG.rst`` file is created as a copy of the package's
+README file.
+
+By default, when the package is available as a platform-independent
+wheel, the ``sage --package`` creates a ``wheel`` package. In this case,
+the ``dependencies`` file is automatically generated from the information
+on PyPI, but may still need some manual editing.
+
+For ``normal`` and ``pip`` packages, the ``dependencies`` file is initialized
+to the bare minimum and will need manual editing. (Watch out for warnings
+regarding ``--no-deps`` that Sage issues during installation of the package!).
 
```

Comment:
at the end, `package!)` without a period.

Reply:
Thanks!